ROMFS/px4fmu_common/init.d/rc.rover_apps

```diff
@@ -14,6 +14,12 @@ then
   rover_interface start
 fi
 
+if param compare SYS_AUTOSTART 50006
```

Comment:
Is it possible to do both of these directly from the rover airframes? It is executed a bit before, but would be cleaner place than doing it here.

Reply:
Another, cleaner option, if you wish, is to set some variable (USE_ROVER_INTERFACE) in the airframe, and here just check that flag and start can0 and rover_interface. This way it is not duplicated for every rover frame.

Reply:
Or, you can use RI_ROVER_TYPE parameter to check these. Just set the RI_ROVER_TYPE default to -1 in rover_interface_params.c, and here initialize rover_interface if the parameter >= 0 ?

Reply:
yes, i will do 
```
if param greater RI_ROVER_TYPE -1
then
  ifup can0
  rover_interface start
fi
```
now, testing again